BundleURL.swift

```diff
@@ -3,30 +3,40 @@ import CoreML
 @available(macOS 10.13, iOS 11.0, tvOS 11.0, watchOS 4.0, *)
 //model name : "FindFour.mlmodelc"
 struct BundleURL{
-    static func bundleUrl(modelName: String) -> URL?{
-        var modelcFile: URL
-        if let documentDirectory =
-            try? FileManager.default.url(for: .documentDirectory, in: .userDomainMask, appropriateFor:nil, create:false) {
-                modelcFile = documentDirectory.appendingPathComponent(modelName)
-            }
+    static func bundleUrl(forResource: String, withExtension: String, modelName: String) -> URL?{
```

Comment:
Minor comment, but I'd make `forResource` and `withExtension` the public variable names with something like `resource` and `extension` internally